testing/trino-testing/src/main/java/io/trino/testing/TestingTrinoClient.java

```diff
@@ -108,7 +108,12 @@ public TestingTrinoClient(TestingTrinoServer trinoServer, Session defaultSession
 
     public TestingTrinoClient(TestingTrinoServer trinoServer, Session defaultSession, OkHttpClient httpClient)
     {
-        super(trinoServer, defaultSession, httpClient);
+        super(trinoServer, defaultSession, httpClient, Optional.empty());
+    }
+
+    public TestingTrinoClient(TestingTrinoServer trinoServer, Session defaultSession, Set<String> clientCapabilities)
+    {
+        super(trinoServer, defaultSession, new OkHttpClient(), Optional.of(clientCapabilities));
```

Comment:
Now the logic how to create new httpclient is spread across AbstractTestingTrinoClient and TestingTrinoClient
would it be possible to avoid that?